### PR TITLE
[visq] Add MSE metric

### PR DESCRIPTION
**Pull request by jinevening**
This adds Mean Squared Error metric.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/9694
Draft PR: https://github.com/Samsung/ONE/pull/9868

### CODE REVIEW

File: compiler/visq/visq [hunk 1]
```diff
@@ -198,6 +203,18 @@ def main():
             with open(args.mpeir_output, "w") as f:
                 json.dump(result, f)
 
+        if args.mse_output:
+            computer = MSEComputer(fp32_dir, fq_dir)
+            qerror_map, q_min, q_max = computer.run()
+            palette = YLORRD9Palette(qerror_min=q_min, qerror_max=q_max)
+            result = _build_json(
+                metric='MSE',
+                model=fp32_model,
```
Comment (mhs4670go) [inline compiler/visq/visq:212]: it is not `Path(fp32_model).stem`?

Comment (seanshpark) [reply compiler/visq/visq:212]: viewer assumes model path is relative to this json or absolute path is also ok.

Comment (mhs4670go) [reply compiler/visq/visq:212]: I got it but above `mpeir_output` case uses `Path(fp32_model).stem`. Is there any difference?

Comment (jinevening) [reply compiler/visq/visq]: Ah, I misunderstood the spec of meta section. Actually it should be `Path(..).name`. Ex: "test.circle"

Path(fp32_model).name -> "test.name"
Path(fp32_model).stem -> "test"

stem will be updated in the next PR. Thanks for finding!

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**